docs/faq.md

```diff
@@ -1,5 +1,42 @@
+## Table of Contents
+1. [Volume Attachment Capacity Issues](#volume-attachment-capacity-issues)
+   - [What causes this misalignment?](#what-causes-this-misalignment)
+   - [What steps can be taken to mitigate this issue?](#what-steps-can-be-taken-to-mitigate-this-issue)
+2. [6-Minute Delays in Attaching Volumes](#6-minute-delays-in-attaching-volumes)
+   - [What causes 6-minute delays in attaching volumes?](#what-causes-6-minute-delays-in-attaching-volumes)
+   - [What behaviors have been observed that contribute to this issue?](#what-behaviors-have-been-observed-that-contribute-to-this-issue)
+   - [What are the symptoms of this issue?](#what-are-the-symptoms-of-this-issue)
+   - [What steps can be taken to mitigate this issue?](#what-steps-can-be-taken-to-mitigate-this-issue-1)
+   - [What is the PreStop lifecycle hook?](#what-is-the-prestop-lifecycle-hook)
+   - [How do I enable the PreStop lifecycle hook?](#how-do-i-enable-the-prestop-lifecycle-hook)
+3. [Driver performance for large-scale clusters](#driver-performance-for-large-scale-clusters)
+   - [Summary of scalability-related changes in v1.25](#summary-of-scalability-related-changes-in-v125)
+   - [EC2 and K8s CSI Sidecar Throttling Overview](#ec2-and-k8s-csi-sidecar-throttling-overview)
+   - [Fine-tuning CSI sidecar scalability parameters](#fine-tuning-csi-sidecar-scalability-parameters)
+4. [IMDSv2 Support](#imdsv2-support)
+5. [CreateVolume (`StorageClass`) Parameters](#createvolume-storageclass-parameters)
+   - [`ext4BigAlloc` and `ext4ClusterSize`](#ext4bigalloc-and-ext4clustersize)
```

Comment:
Given that GitHub [already has a built in table of contents](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/) that is automatically generated, this can probably be removed, example from the live repo:

![image](https://github.com/user-attachments/assets/80e67ebd-b713-4d9d-a9af-a1f3845b2bed)